code/modules/food_and_drinks/recipes/food_mixtures.dm

```diff
@@ -184,3 +184,9 @@
 	results = list(/datum/reagent/consumable/nutriment/peptides = 0.5)
 	required_reagents = list(/datum/reagent/consumable/nutriment/ = 0.5)
 	required_catalysts = list(/datum/reagent/medicine/metafactor = 0.5)
+
+/datum/chemical_reaction/bbqsauce
+	name = "BBQ Sauce"
+	id = /datum/reagent/consumable/bbqsauce
+	results = list(/datum/reagent/consumable/bbqsauce = 20)
+	required_reagents = list(/datum/reagent/consumable/sugar = 5, /datum/reagent/ash = 5, /datum/reagent/consumable/capsaicin = 5, /datum/reagent/water = 5)
```

Comment:
Can't believe I'm bikeshedding about BBQ Sauce in a 2D clown abuse simulator, but..
Can you add salt and black pepper to the recipe? The lack of vinegar hurts me, but we just don't have that reagent in the codebase. Also, what's the ash for?

Reply:
Smokiness silly.

Reply:
Ah yeah that works

Reply:
More than doable, sure.